src/testrender/cuda/vec_math.h

```diff
@@ -0,0 +1,97 @@
+// Copyright Contributors to the Open Shading Language project.
+// SPDX-License-Identifier: BSD-3-Clause
```

Comment:
I thought Imath now supported cuda out of the box. Is this still needed?

Reply:
Only the casting macros are needed in the current iteration. So I'll move them to where they are used and remove this file.